src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayController.cs

```diff
@@ -135,14 +210,14 @@ public async Task<IActionResult> Evaluated(Guid applicationId)
         [HttpGet("/Roatp/GatewayCheckStatus/{applicationId}/Page/{PageId}/Status/{gatewayReviewStatus}")]
         public async Task<IActionResult> CheckStatus(Guid applicationId, string PageId, string gatewayReviewStatus)
         {
-            if (gatewayReviewStatus.Equals(GatewayReviewStatus.New)) 
+            if (gatewayReviewStatus.Equals(GatewayReviewStatus.New))
             {
                 var username = _contextAccessor.HttpContext.User.UserDisplayName();
                 await _applyApiClient.TriggerGatewayDataGathering(applicationId, username);
                 await _applyApiClient.StartGatewayReview(applicationId, _contextAccessor.HttpContext.User.UserDisplayName());
             }
 
-            return Redirect($"/Roatp/Gateway/{applicationId}/Page/{PageId}"); 
+            return Redirect($"/Roatp/Gateway/{applicationId}/Page/{PageId}");
```

Comment:
Can this be a RedirectToAction please?

Reply:
Depending on the different PageIds it goes to different actions, even different controllers.
Although this bit of code is not really part of this PR I will change it, but I cannot see more suitable method of resolving the redirecting. And definitely not RedirectToAction.

Reply:
Ok, that one is ok to leave then, if it maps to multiple controllers